sys-libs/tevent/tevent-0.10.2.ebuild

```diff
@@ -1,43 +1,48 @@
 # Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
+
 PYTHON_COMPAT=( python3_{6..9} )
 PYTHON_REQ_USE="threads(+)"
-
 inherit waf-utils multilib-minimal python-single-r1
 
 DESCRIPTION="Samba tevent library"
 HOMEPAGE="https://tevent.samba.org/"
-SRC_URI="https://www.samba.org/ftp/tevent/${P}.tar.gz"
+SRC_URI="https://samba.org/ftp/tevent/${P}.tar.gz"
 
 LICENSE="GPL-3"
 SLOT="0"
 KEYWORDS="~alpha amd64 arm arm64 hppa ~ia64 ~m68k ~mips ppc ppc64 ~riscv ~s390 sparc x86 ~x86-linux"
 IUSE="elibc_glibc python"
 
-RDEPEND="!elibc_FreeBSD? ( dev-libs/libbsd[${MULTILIB_USEDEP}] )
-	>=sys-libs/talloc-2.3.1[python?,${PYTHON_SINGLE_USEDEP},${MULTILIB_USEDEP}]
-	python? ( ${PYTHON_DEPS} )"
+# build system does not work with python3
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+RESTRICT="test"
 
+RDEPEND="
+	!elibc_FreeBSD? ( dev-libs/libbsd[${MULTILIB_USEDEP}] )
+	>=sys-libs/talloc-2.3.1[${MULTILIB_USEDEP}]
+	python? (
+		${PYTHON_DEPS}
+		sys-libs/talloc[python,${PYTHON_SINGLE_USEDEP}]
+	)
+"
 DEPEND="${RDEPEND}
-	virtual/pkgconfig
 	elibc_glibc? (
 		net-libs/libtirpc[${MULTILIB_USEDEP}]
 		|| (
 			net-libs/rpcsvc-proto
 			<sys-libs/glibc-2.26[rpc(+)]
 		)
 	)
-	${PYTHON_DEPS}
 "
-# build system does not work with python3
```

Comment:
that comment is probably also outdated ;)

Reply:
Yeah, drop it please.